src/sentry/models/commitfilechange.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+)
+
+
+class CommitFileChange(Model):
+    __core__ = False
+
+    organization_id = BoundedPositiveIntegerField(db_index=True)
+    commit = FlexibleForeignKey('sentry.Commit')
+    filename = models.CharField(max_length=255)
+    type = models.CharField(max_length=1, choices=(
```

Comment:
Can we just use an integer here? Unless I'm missing something, it seems weird to just use a CharField, with a max_length of 1, when we can just use a smallint. Should be insignificant in practice, just feels less error prone to use a enum like we do elsewhere, like, `CommitType.ADDED`, etc.